admin/ajax.php

```diff
@@ -359,3 +359,6 @@ function wpseo_add_fb_admin() {
 }
 
 add_action( 'wp_ajax_wpseo_add_fb_admin', 'wpseo_add_fb_admin' );
+
+// Crawl Issue Manager AJAX hooks.
+new WPSEO_GWT_Ajax;
```

Comment:
Interesting point! But for now we will not support hacking our filters and actions until someone comes up with a generic usecase for that.